.validate-branch-namerc.js

```diff
@@ -0,0 +1,11 @@
+const BRANCH_REGEX =
```

Comment:
Should this be a dotfile? Also typically files that end in `rc` are config files, but this is a script. Wondering if it makes sense to move this to the `development` directory and call this `validate-branch-name.js`.